comparison/loadtest.go

```diff
@@ -168,6 +171,10 @@ func buildLoadDBDumpCmds(dumpFilename string, newIP string, permalinkIPsToReplac
 		return []string{}, fmt.Errorf("invalid db engine %s", dbInfo.Engine)
 	}
 
+	if sedCmd == "" {
```

Comment:
Nit: Maybe invert the condition and check for `sedCmd != ""`? Feels more natural to me that way.

Reply:
-1/5, so just inverted it :)